docs/manual/introduction.tex

```diff
@@ -721,6 +721,10 @@
   JDK, but could also be useful when the warning/note is redundant for a
   user.
   See comments for \<-ApermitUnsupportedJdkVersion> option above.
+\item \<-AcheckRedundantAnnotations>
```

Comment:
This seems like the best section for this new option, but I would move it earlier in the list, probably as the second one, between `-Alint` and `-AsuggestPureMethods`.